shell/platform/android/io/flutter/embedding/engine/plugins/FlutterPlugin.java

```diff
@@ -93,10 +95,10 @@
    * concerned about which Android Component is currently holding the {@link FlutterEngine}.
    * TODO(mattcarroll): add info about ActivityAware and ServiceAware for plugins that care.
    */
-  class FlutterPluginBinding implements LifecycleOwner {
+  class FlutterPluginBinding {
         private final Context applicationContext;
         private final FlutterEngine flutterEngine;
-        private final Lifecycle lifecycle;
```

Comment:
ditto

Reply:
I'll resolve this issue since it's a dup of the conversation above.